opennms-container/horizon/CONFD_README.md

```diff
@@ -0,0 +1,26 @@
+# Configuring Horizon via confd
+(instructions for testing/developing confd templates are given at the end of this document)
+## Mounting
+When starting the Horizon container, mount a yaml file to the following path `/opt/minion/horizon-config.yaml`.
+
+Any configuration provided to confd will overwrite configuration specified as environment variables. Direct overlay of
+specific configuration files will overwrite the corresponding config provided by confd.
+
+## Contents
+The following describes the keys that can be specified in `horizon-config.yaml` to configure Horizon via confd.
+
+### Slack
+
+```
+--- 
+notifd:
```

Comment:
The slack path is `$slackPath := "/opennms/notifd/slack/"`. Should it be then:
```
---
opennms:
  notifd:
    slack:
      channel: ...
```

Reply:
Looks like it should be.

Reply:
That's my fault. I just copied the notifd part out of the file. Sorry for the confusion.
It looks like this:

```
mf1045@mf1045-nb115:~/Dokumente/git/onms/stack-play/minimal-horizon/container-fs/opt/opennms-overlay/confd$ cat horizon-config.yaml 
---
postgres:
  host: database
  port: 5432
  user: postgres
  password: postgres

opennms:
  dbname: opennms
  dbuser: opennms
  dbpass: opennms

  timeseries:
    strategy: rrd

  rrd:
    storebyforeignsource: true
    strategyclass: org.opennms.netmgt.rrd.rrdtool.MultithreadedJniRrdStrategy

  library:
    jrrd2: /usr/lib64/libjrrd2.so

  notifd:
    slack:
      channel: Alerting
      userName: AlertingUser
      iconEmoji: AlertingIcon
      iconURL: AlertingURL
```